modules/loadbalancer/templates/nginx_balance.conf.erb

```diff
@@ -52,6 +52,12 @@ server {
   # Try next upstream if one returns an error
   proxy_next_upstream error http_500;
```

Comment:
I wonder if we should move this into an `else` block (if that's even possible in Nginx?) to avoid someone seeing that line and missing the bit below it.

Reply:
Unfortunately you can't do `else` in nginx, you just need loads of consecutive `if` statements.